README.md

```diff
@@ -9,6 +9,8 @@ bktec supports multiple test runners and offers various features to enhance your
 | Filter test files                                  |   ✅  |   ✅  |    ✅      |    ✅   |
 | Automatically retry failed test                    |   ✅  |   ✅  |    ✅      |    ❌   |
 | Split slow files by individual test example        |   ✅  |   ❌  |    ❌      |    ❌   |
+| Mute tests (ignore tests failure)                  |   ✅  |   ✅  |    ✅      |    ❌   |
```

Comment:
Grammatically I think this needs to be `ignore test failure` or `ignore tests' failure`

Reply:
Actually maybe its `ignore test failures`?